shared/constants/form.ts

```diff
@@ -51,3 +51,5 @@ export const ADMIN_FORM_META_FIELDS = <const>[
   '_id',
   'responseMode',
 ]
+
+export const PAYMENT_CONTACT_FIELD_ID = 'payment_contact_field'
```

Comment:
for consistency, let's stick to one term for each type of entity. I would recommend `FormFields` and `PaymentField`. You could also pick `PaymentContactField`, as long as we're consistent throughout the verification module. These will need to be made consistent in various places - the `model`, `types`, `errors` and `util` files, to name a few.

Reply:
With the refactor, there are no more form or payment specific methods as paths are distinguished by the `fieldId` or `isPayment` flag.